documentation/modules/exploit/windows/smb/smb_relay.md

```diff
@@ -0,0 +1,301 @@
+## Vulnerable Application
+
+This module supports running an SMB server which validates credentials, and then attempts to
+execute a relay attack against the configured RELAY_TARGETS hosts.
+
+Supports SMBv2, SMBv3, and captures NTLMv1 as well as NTLMv2 hashes.
+SMBv1 is not supported - please see https://github.com/rapid7/metasploit-framework/issues/16261
+
+## Verification Steps
+
+### Machine setup
+
+Note, for some targets you may require configuring `LocalAccountTokenFilterPolicy` to allow psexec:
+```
+reg add HKLM\Software\Microsoft\Windows\CurrentVersion\Policies\System /v LocalAccountTokenFilterPolicy /t REG_DWORD /d 1 /f
+```
+
+Or uac, which requires a restart:
+```
+reg add HKLM\Software\Microsoft\Windows\CurrentVersion\Policies\System /v EnableLUA /t REG_DWORD /d 0 /f
+```
+
+You may need to change the policy on hosts that you are relaying to https://docs.microsoft.com/en-us/troubleshoot/windows-server/networking/overview-server-message-block-signing#policy-locations-for-smb-signing
+
+This can be checked in powershell with:
+
+```
+PS > reg query HKLM\SYSTEM\CurrentControlSet\Services\LanmanServer\Parameters /v requiresecuritysignature
+
+HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\LanmanServer\Parameters
+    requiresecuritysignature    REG_DWORD    0x0
+```
+
+Set the registry key value with:
+```
+reg add HKLM\SYSTEM\CurrentControlSet\Services\LanmanServer\Parameters /v requiresecuritysignature /t REG_DWORD /d 0 /f
+```
+
+The above also works on domain controllers, but you may also have to modify a group policy settings.
+
+#### Lab setup
+
+Example relaying between:
+
+- Windows 7 <-> Metasploit framework <-> Windows 10
+
+Where windows 7 = 192.168.123.22
+Where msfconsole = 192.168.123.1
+Where windows 10 = 192.168.123.4
+
+Example lab diagram:
+
+```mermaid
+flowchart LR
+    A("A\nWindows 7\n192.168.123.22")
+    subgraph metasploit["   B - msfconsole - 192.168.123.1 "]
+        subgraph inside [ ]
+            direction TB
+            style inside margin-top: 0
+            style inside stroke: none
+
+            B("smb_relay")
+            database[(Database)]
+
+            B -->|"report_ntlm_type3(...)"| database
+        end
+    end
+    C("C\nWindows 10\n 192.168.123.4")
+
+    A <-->|SMB 445| metasploit
+    metasploit <-->|"smb 445\npsexec"| C
+```
+
+## Options
+
+### RELAY_TARGETS
+
+Target address range or CIDR identifier to relay to
+
+### CAINPWFILE
+
+A file to store Cain & Abel formatted captured hashes in. Only supports NTLMv1 Hashes.
+
+### JOHNPWFILE
+
+A file to store John the Ripper formatted hashes in. NTLMv1 and NTLMv2 hashes will be stored in separate files.
+I.E. the filename john will produce two files, `john_netntlm` and `john_netntlmv2`.
```

Comment:
This is weird existing behavior IMO, I'll be throwing up a separate PR to remove this